packages/core/docs/i18n.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Foundation/Internationalization" />
+
+# Internationalization (i18n)
+
+Clarity Core provides an i18n service to change the internal component strings for various region support easily. Clarity Core i18n can be controlled in two different ways.
+
+1. Global Scope
+2. Component Scope
+
+## Global Scope
+
+Global Scope i18n uses the `I18nService` service from `@cds/core`. The service enables the ability to override the default component strings as well as read them.
```

Comment:
`i18n in the global scope uses the `I18nService` service from `@cds/core`. This service can both read and override the default component strings.`

instead of?

`Global Scope i18n uses the `I18nService` service from `@cds/core`. The service enables the ability to override the default component strings as well as read them.`